src/plugins/intel_npu/src/al/include/intel_npu/config/npuw.hpp

```diff
@@ -247,6 +247,36 @@ struct NPUW_LLM_PREFILL_CONFIG final : OptionBase<NPUW_LLM_PREFILL_CONFIG, ov::A
     }
 };
 
+struct NPUW_LLM_ADDITIONAL_PREFILL_CONFIG final : OptionBase<NPUW_LLM_ADDITIONAL_PREFILL_CONFIG, ov::AnyMap> {
+    static std::string_view key() {
```

Comment:
What is this, why

Reply:
To support caching for any additional configs we might pass